client/src/components/content/RobotContainer/CreateRobotContainer.jsx

```diff
@@ -35,3 +36,7 @@ const CreateRobotContainer = () => {
     )
 };
 export default CreateRobotContainer;
+
+CreateRobotContainer.propTypes = {
+    createNewRobot: PropTypes.string.isRequired,
+};
```

Comment:
Export should start after propTypes :) I just looked it up on reactjs.org and found this example:
![grafik](https://user-images.githubusercontent.com/42138601/110779807-47549c80-8264-11eb-8390-d2a5738cc6b8.png)

Reply:
Please change this for all the documents where you used propTypes

Reply:
Ahh :D just read now. Awesome research, thank you! Had it already changed anyway.